activeadmin.gemspec

```diff
@@ -17,16 +17,16 @@ Gem::Specification.new do |s|
 
   s.extra_rdoc_files = %w[CHANGELOG.md CODE_OF_CONDUCT.md CONTRIBUTING.md README.md]
 
-  s.required_ruby_version = '>= 2.4'
+  s.required_ruby_version = '>= 2.5'
 
   s.add_dependency 'arbre', '~> 1.2', '>= 1.2.1'
-  s.add_dependency 'formtastic', '~> 3.1'
-  s.add_dependency 'formtastic_i18n', '~> 0.4'
-  s.add_dependency 'inherited_resources', '~> 1.7'
-  s.add_dependency 'jquery-rails', '~> 4.2'
-  s.add_dependency 'kaminari', '~> 1.0', '>= 1.0.1'
+  s.add_dependency 'formtastic', '>= 3.1'
+  s.add_dependency 'formtastic_i18n', '>= 0.4'
+  s.add_dependency 'inherited_resources', '>= 1.7'
+  s.add_dependency 'jquery-rails', '>= 1.0'
+  s.add_dependency 'kaminari', '~> 1.0', '>= 1.1.1'
```

Comment:
In this case, you're bumping the minimum version of `kaminari`, efectively dropping support for versions between 1.0.1 and 1.1.1. In general, to be as flexible as possible with the range of dependencies we support, we don't do that unless we have a reason.

Reply:
Same reason as before, leaving a very wide open range of gem acceptance was not doing the trick the resolver would fall in the lowest and break the rest of the bundle.

Reply:
I think we made this change, or similar, separately, so it should no longer be a problem.